creator-node/src/serviceRegistry.js

```diff
@@ -193,6 +194,68 @@ class ServiceRegistry {
     app.use('/health/bull', serverAdapter.getRouter())
   }
 
+  /**
+   * Initializes the app with duration tracking metrics
+   * @param {Object} app
+   */
+  async _setupRouteDurationTracking(app) {
```

Comment:
if you found this code we should add a link. if you wrote this, it's impressive but also we should add a few more comments?